.github/workflows/clang-tools-amd64.yml

```diff
@@ -75,6 +75,11 @@ jobs:
             shacmd: 'sha512sum.exe'
             extra-tar-args: '--exclude=${RELEASE}/clang/test/Driver/Inputs/* --exclude=${RELEASE}/libcxx/test/std/input.output/filesystems/Inputs/static_test_env/* --exclude=${RELEASE}/libclc/amdgcn-mesa3d'
             extra-tar-args-cfe: '--exclude=cfe-${version}.src/test/Driver/Inputs/*'
+        exclude:
+          # TODO: macosx clang 18 is failing.  Example: https://github.com/colinsullivan/clang-tools-static-binaries/actions/runs/14976168180/job/42068957048
```

Comment:
There was also a linker error above for 18 on macos - I could simply retry the build - I suspect the mac VM needs to be updated too...

https://github.com/colinsullivan/clang-tools-static-binaries/actions/runs/14976168180/job/42068957048

CC @shenxianpeng 

Could also try to fix this before merging.

Reply:
thank you , yea let's get it working before merging here.